content/python/concepts/file-methods/terms/remove/remove.md

```diff
@@ -0,0 +1,46 @@
+---
+Title: '.remove()'
+Description: 'Allows the user to delete a file.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Methods'
+  - 'Files'
+  - 'Functions'
+  - 'Strings'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The Python `.remove()` file method allows the user to delete a file if it exists in the system or computer. This method is provided by the `os` module which must be imported beforehand. [`.unlink()`](https://www.codecademy.com/resources/docs/python/file-methods/unlink) is an alternative to delete a file.
```

Comment:
```suggestion
The `.remove()` file method allows the user to delete a file if it exists in the system or computer. This method is provided by the `os` module which must be imported beforehand. [`.unlink()`](https://www.codecademy.com/resources/docs/python/file-methods/unlink) is an alternative to delete a file.
```

Reply:
> This method is provided by the `os` module which must be imported beforehand.

I'm wondering if this sentence should be moved somewhere in the Syntax section? 🤔

Reply:
Moving it in the next push!